runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotConfig.java

```diff
@@ -143,11 +143,22 @@ default boolean screenshotBossKills()
 		return false;
 	}
 
+	@ConfigItem(
+		keyName = "playerDeads",
+		name = "Screenshot Deaths",
+		description = "Configures whether or not screenshots are automatically taken when you die.",
+		position = 10
+	)
+	default boolean screenshotPlayerDeads()
```

Comment:
`screenshotPlayerDeaths` sounds a little better to me, but we can see what other people think.

(same goes for the key name: `playerDeaths`)

Reply:
Why not make it singular, the plurality sounds weird since it only applies to yourself.
I do definitely support the change from dead to death as well.

Reply:
Nice catch @Alexsuperfly , I agree with you.